frontend/app/module/input/page/FinaliseElectionPage.tsx

```diff
@@ -0,0 +1,61 @@
+import { Link } from "react-router-dom";
+
+import { NavBar } from "app/component/navbar/NavBar.tsx";
+
+import { useElection, useElectionStatus } from "@kiesraad/api";
+import { IconChevronRight } from "@kiesraad/icon";
+import { Button, PageTitle, WorkStationNumber } from "@kiesraad/ui";
+
+export function FinaliseElectionPage() {
+  const { election } = useElection();
+  const { statuses } = useElectionStatus();
+
+  // Safeguard so users cannot circumvent the check via the browser's address bar
+  if (statuses.some((s) => s.status !== "Complete")) {
+    throw Error("election not ready for finalisation");
```

Comment:
Is this only an error for the console or does this show to the user? If the latter is true there should be a text in Dutch maybe?

Reply:
This is an error in the console. It is thrown as Error, so the user will see the NotFound page. The error will trigger a render of the `errorElement` attribute defined in the relevant `<Route>` (or one of its parents) in `routes.tsx`.

Reply:
I've created #277 for improving the user-visible part of the error handling (there's a nice page being designed).

This console error should be fine for now.